### ISSUE TITLE
Cannot read property '_' of undefined

### ISSUE THREAD

**Issue by brettwray** [feathers-plus/feathers-vuex#248]
### Steps to reproduce
Followed the installation instructions in the documentation, the project is not using Vue-Cli, but it is set up with webpack and everything prior to adding feathers-vuex was working correctly.

In my webpack babel loader I have the following rule as suggested in the feathers docs: exclude: /node_modules(\/|\\)(?!(@feathersjs|debug))/

Webpack is using webpack dev server and hmr.

Feathers server is running on port 3030, but the feathers app is in another directory.

### Expected behavior

Front end should load, Vue app should work without throwing error.

### Actual behavior
Error in the console: 

> Uncaught TypeError: Cannot read property '_' of undefined
>  at eval (getters.js?b50b:28)

getters.js line 28: var _ = _commons2.default._;

### System configuration

Webpack with webpack dev server, HMR.

Babel-loader, plugins: plugin-syntax-dynamic-import, plugin-proposal-class-properties.

Front end: Vue with webpack dev server running on localhost:8080

Back-end: Feathers server running on localhost:3030

**Module versions** (especially the part that's not working): "feathers-vuex": "^1.7.0",

**NodeJS version**: 10.16.0

**Operating System**: OSX Mojave

**Browser Version**: Chrome 76

**Vuex Version**:3.1.1

**Module Loader**: webpack 4.39.2

**feathers-client.js:** 

> import feathers from '@feathersjs/feathers'
> import socketio from '@feathersjs/socketio-client'
> import auth from '@feathersjs/authentication-client'
> import io from 'socket.io-client'
> 
> const socket = io('http://localhost:3030', {  transports : ['websocket']});
> 
> const feathersClient = feathers()
>       .configure(socketio(socket))
>       .configure(auth({storage: window.localStorage}));
> 
> export default feathersClient;

**store/index.js:**

> import Vue from 'vue'
> import Vuex from 'vuex'
> import feathersVuex from 'feathers-vuex'
> import feathersClient from '../../config/feathers-client'
> 
> const { service, auth, FeathersVuex } = feathersVuex(feathersClient, {  idField: '_id'});
> 
> 
> Vue.use(Vuex)
> Vue.use(FeathersVuex)
> // MODULES
> import baseState from './modules/base';
> import session from './modules/session';
> 
> export default new Vuex.Store({
>   plugins: [
>     service({}),
>     auth({  userService : 'users'})
>   ],
>   modules: {
>     baseState,
>     session
>   }
> })

**Comment by adrianofoschi** [comment]
I opened issue for the same error
https://github.com/feathers-plus/feathers-vuex/issues/247

**Comment by enderson-pan** [comment]
The same issue!

It's ok with @feathersjs/commons 4.0.0，however, failed with  @feathersjs/commons 4.3.0

Feathers-vuex 1.7.0:
<img width="815" alt="image" src="https://user-images.githubusercontent.com/1694834/63873378-0f736480-c9f2-11e9-93ed-da372cb6e063.png">
<img width="1038" alt="image" src="https://user-images.githubusercontent.com/1694834/63873412-21550780-c9f2-11e9-9a88-b9b0095bbf60.png">

Maybe because of(with @feathersjs/commons 4.3.0):
<img width="439" alt="image" src="https://user-images.githubusercontent.com/1694834/63873567-6da04780-c9f2-11e9-8f95-072189e43fac.png">
![image](https://user-images.githubusercontent.com/1694834/63873651-8f99ca00-c9f2-11e9-96dc-ecafa02684be.png)
![image](https://user-images.githubusercontent.com/1694834/63873695-a17b6d00-c9f2-11e9-8b36-81e3f09abf1c.png)

The correct one(with @feathersjs/commons 4.0.0):
![image](https://user-images.githubusercontent.com/1694834/63873780-c53eb300-c9f2-11e9-9b68-fd835c42912e.png)


It seems like an ES6 module loads problem.

Because @feathers/commons upgrades to 4.3.0

Change 
"@feathersjs/commons": "^4.0.0",
to
"@feathersjs/commons": "4.0.0",
 in feathersjs-vuex's package.js may fix the problem.

**Comment by adrianofoschi** [comment]
I'm using feathers 4.3.0 and feathers-vuex 2.0. But the problem is the same in your version.
I **solved** replacing in service-modules.getters.js
```
import commons from '@feathersjs/commons'
const _ = commons;
```
with
```
import { _ } from '@feathersjs/commons'
```

I think it needs an hotfix.

**Comment by marshallswain** [comment]
I ran into this briefly, today, but it went away after a reinstall.  I'll look into it again when I can reproduce it.

**Comment by brettwray** [comment]
The issue appears to be with _commons2.**default**._.;

By removing **.default** I was able to clear that error, but it remains to be seen if that will just cause future problems.

![Screen Shot 2019-08-28 at 8 38 02 PM](https://user-images.githubusercontent.com/13531627/63908445-0378aa00-c9d4-11e9-9de7-248273c0451b.png)

**Comment by marshallswain** [comment]
I've released `feathers-vuex@2.0.0-pre.75` with the import fixed.  Please let me know how it goes.

**Comment by CheyenneForbes** [comment]
@marshallswain I installed `feathers-vuex 2.0.0-pre.75` and now I'm getting an error:
```
node_modules/feathers-vuex/dist/index.js:6
import setupAuthModule from './auth-module/auth-module'
       ^^^^^^^^^^^^^^^

SyntaxError: Unexpected identifier
```

**Comment by marshallswain** [comment]
@CheyenneForbes make sure you add `feathers-vuex` to your vue.config file in the `transpileDependencies` section.

**Comment by dkebler** [comment]
Yup having same issue which has me scratching my head cause I have a deployed system using 1.7 back in April but now when I go to start a clean install this appears. All I can figure is that somewhere in 3.0 feathersjs did what amounted to a breaking change by removing the default export which might have been about the time they moved commons inside the main repo.  I assume they could do that cause they figured no one would try to load commons by directly (ha).  

@marshallswain 
Does 2.0 support feathersjs 3.0 or must I move to 4.0?   Is there a fix/patch for 1.7 so I can continue to use it instead of upgrading to the yet to be released 2.0
